golem-service-base/src/storage/blob/s3.rs

```diff
@@ -254,7 +253,11 @@ impl S3BlobStorage {
             SdkError::ConstructionFailure(_) => "Construction failure".to_string(),
             SdkError::TimeoutError(_) => "Timeout".to_string(),
             SdkError::DispatchFailure(inner) => {
-                format!("Dispatch failure: {}", inner.as_connector_error().unwrap())
+                // normal display of the error does not expose enough useful information
```

Comment:
Please fix all error mapping to have logs. Our open issue in prod is about contruction failure.